hail/python/hail/experimental/vcf_combiner/vds_combiner.py

```diff
@@ -0,0 +1,190 @@
+from typing import List
+
+import hail as hl
+from hail import MatrixTable, Table
+from hail.ir import Apply, TableMapRows, TopLevelReference
+from hail.typecheck import oneof, sequenceof, typecheck
+from hail.vds import VariantDataset
+from .vcf_combiner import combine_gvcfs, localize, parse_as_fields, unlocalize
+
+_transform_variant_function_map = {}
+_transform_reference_fuction_map = {}
+
+
+def make_variants_table(mt, info_to_keep=[]) -> Table:
+    mt = mt.filter(hl.is_missing(mt.info.END))
+
+    if mt.row.dtype not in _transform_variant_function_map:
+        def get_lgt(e, n_alleles, has_non_ref, row):
```

Comment:
General thought, but we should make a scode function for this. It'll be way better.